docs/index.rst

```diff
@@ -124,87 +124,87 @@ There are two ways to save the WCS to a file:
 
 
-A step by step example of constructing an imaging GWCS object.
+A step-by-step example of constructing an imaging GWCS object.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The following example shows how to construct a GWCS object equivalent to
-a FITS imaging WCS without distortion, defined in this FITS imaging header::
-
-  WCSAXES =                    2 / Number of coordinate axes
-  WCSNAME = '47 Tuc     '        / Coordinate system title
-  CRPIX1  =               2048.0 / Pixel coordinate of reference point
-  CRPIX2  =               1024.0 / Pixel coordinate of reference point
-  PC1_1   =   1.290551569736E-05 / Coordinate transformation matrix element
-  PC1_2   =  5.9525007864732E-06 / Coordinate transformation matrix element
-  PC2_1   =  5.0226382102765E-06 / Coordinate transformation matrix element
-  PC2_2   = -1.2644844123757E-05 / Coordinate transformation matrix element
-  CDELT1  =                  1.0 / [deg] Coordinate increment at reference point
-  CDELT2  =                  1.0 / [deg] Coordinate increment at reference point
-  CUNIT1  = 'deg'                / Units of coordinate increment and value
-  CUNIT2  = 'deg'                / Units of coordinate increment and value
-  CTYPE1  = 'RA---TAN'           / TAN (gnomonic) projection + SIP distortions
-  CTYPE2  = 'DEC--TAN'           / TAN (gnomonic) projection + SIP distortions
-  CRVAL1  =        5.63056810618 / [deg] Coordinate value at reference point
-  CRVAL2  =      -72.05457184279 / [deg] Coordinate value at reference point
-  LONPOLE =                180.0 / [deg] Native longitude of celestial pole
-  LATPOLE =      -72.05457184279 / [deg] Native latitude of celestial pole
-  RADESYS = 'ICRS'                / Equatorial coordinate system
+The following example shows how to construct a GWCS object that maps
+input pixel coordinates to sky coordinates. This example 
+involves 4 sequential transformations: 
 
+- Adjusting pixel coordinates such that the center of the array has 
+  (0, 0) value (typical of most WCS definitions, but any pixel may
+  be the reference that is tied to the sky reference, even the (0, 0)
+  pixel, or even pixels outside of the detector).
+- Scaling pixels such that the center pixel of the array has the expected
+  angular scale. (I.e., applying the plate scale)
+- Projecting the resultant coordinates onto the sky using the tangent
+  projection. If the field of view is small, the inaccuracies resulting
+  leaving this out will be small; however, this is generally applied.
+- Transforming the center pixel to the appropriate celestial coordinate 
+  with the approprate orientation on the sky. For simplicity's sake,
+  we assume the detector array is already oriented with north up, and
+  that the array has the appropriate parity as the sky coordinates.
+
+
+The detector has a 1000 pixel by 1000 pixel array.
+
+For simplicity, no units will be used, but instead will be implicit.
```

Comment:
shouldn't this go into the example in `fits_analog.rst`?

Reply:
Again, do you mean duplicating (with needed changes) it there rather than moving?